examples/logger.py

```diff
@@ -54,6 +56,7 @@ class LoggerFrame(QWidget):
     Attributes:
         logEdit: A textEdit which shows all logs.
         clearButton: A button for clearing all logs.
+        setLevelBox: A combo box for setting the logger's level.
```

Comment:
It's a minor suggestion, but how about changing 'combo box' into 'comboBox' similar to `logEdit`?